<launch>
  <arg name="model" />
  <param name="robot_description"
    textfile="$(find cobod_arm)/urdf/cobod_arm.urdf" />
  <!-- <node 
    name="joint_state_publisher_gui"
    pkg="joint_state_publisher_gui"
    type="joint_state_publisher_gui" />-->
  <node
    name="robot_state_publisher"
    pkg="robot_state_publisher"
    type="robot_state_publisher" /> 
  <node
    name="rviz"
    pkg="rviz"
    type="rviz"
    args="-d $(find cobod_arm)/rviz/config_file.rviz" />

  <param name="errorIK" type="double" value="0.05" />
  <param name="timeIK" type="double" value="0.1" />
  <param name="robot_name" type="string" value="cobod" />
  <node pkg="send_pos" name="Follow_traj_trak_ik_cobod" type="Follow_traj_trak_ik_cobod" output="screen"/>
    

</launch>